packages/api/src/external/carequality/command/update-patient-discovery-status.ts

```diff
@@ -0,0 +1,50 @@
+import { Patient } from "@metriport/core/domain/patient";
```

Comment:
Im adding the ability to update the CQ external data for a patient so we can determine from outside of the logic (for script and internal utils) whether or not the patient discovery is processing complete or failed

Reply:
The reasoning is because when we do a PD we do not have access to the requestid because we do not store this anywhere and we also dont store the num of gatways the request is making. Both of them are necessary to determine whether the request is complete (this is what is in the polling function)